parse.go

```diff
@@ -118,6 +136,7 @@ func Parse(fs *flag.FlagSet, args []string, options ...Option) error {
 // Context contains private fields used during parsing.
 type Context struct {
 	configFile             string
```

Comment:
I think you can refactor WithConfigFile to be sugar for WithConfigFileVia, which would let you remove this field and leave Parse essentially untouched.

Reply:
done